build/dappnode/scripts/dappnode_install_pre.sh

```diff
@@ -124,6 +124,28 @@ install_docker_compose() {
     fi
 }
 
+
+install_wireguard_dkms() {
+    ##############################################
+    ##############################################
+    ####        WIREGUARD INSTALLATION        ####
+    ##############################################
+    ##############################################
+    apt-get update -y
```

Comment:
is this command strictly necessary? maybe such critical command should be executed in other place and not in wireguard stuff. 
Also would be interesting to redirect STDERR to a log file

Reply:
It is not `upgrade` command so it is not that critical. But yes, STDERR should be redirected